bh_core.sublime-settings

```diff
@@ -86,16 +86,16 @@
     // Experimental: Special matching mode for block cursor.
     // Essentially, this provides a matching mode that makes a little more
     // sense to some in regards to the visual representation of block cursors.
-    // This will ignore "bracket_outside_adjacent".
+    // This will ignore `bracket_outside_adjacent`.
     "block_cursor_mode": false,
 
-    // When "bracket_outside_adjacet" is set, and a plugin command explicitly sets
-    // "no_outside_adj" to "None" instead of "true" or the default "false",
+    // When `bracket_outside_adjacet` is set, and a plugin command explicitly sets
```

Comment:
Should be `bracket_outside_adjacent` (missing `n`).

Reply:
Thanks for pointing this out.  I had put this on ice for a bit as I wasn't happy with my first implementation of the spelling tool and only recently got around to refactoring.  This got overlooked as I'm specifically avoiding things in back ticks (mainly non-words like function names and settings values etc.).